src/components/TimecodeTracker/TimecodeTracker.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import {
+  getTimecodeGameLength,
+  getTimecodeTotalLength,
+} from '../../utils/Timecode'
+import TimecodeInput from './TimecodeInput'
+import TimecodeResult from './TimecodeResult'
+
+export default function TimecodeTracker() {
+  const [timecodes, setTimecodes] = useState(['', '', ''])
+
+  return (
+    <>
+      <form onKeyDown={handleEnter}>
+        {timecodes.map((_, index) => (
+          <TimecodeInput
+            title={index < 2 ? 'Halbzeit ' + (index + 1) : 'O-Töne'}
+            index={index}
+            key={index}
+            onChange={handleChange}
+          />
+        ))}
+      </form>
+      <TimecodeResult
+        text="Länge Spiel"
+        result={getTimecodeGameLength(timecodes)}
+      />
+      <TimecodeResult
+        text="Länge gesamt"
+        result={getTimecodeTotalLength(timecodes)}
+      />
+    </>
+  )
+
+  function handleChange(event, index) {
+    const value = event.target.value
+    const oldTimecodes = [...timecodes]
+    oldTimecodes[index] = value
+    setTimecodes(oldTimecodes)
+  }
+
+  function handleEnter(event) {
```

Comment:
handleEnter might be correct but as a person seeing this function for the first time I have no idea what it does. So maybe a function name more related to the value of the function?

Reply:
will change for better understanding.